.gitignore

```diff
@@ -0,0 +1,12 @@
+*.js
```

Comment:
I guess you added `*.js` and `*.d.ts` because of how your typescript configuration is generating the JS files. Why not use 
 the [outDir](https://www.typescriptlang.org/tsconfig#outDir) compiler option to put all the compiled files in a single directory?

Reply:
True. This was just the default `gitignore`created when you bootstrap a new typescript project with the CDK. I think the reason they do it this way is because of the js files generated in the `bin` and `lib` folder. But, to your point, I like them being separate. Added dist folder and updated `.gitignore`